internal/pkg/daemon/bpfrecorder/bpfrecorder.go

```diff
@@ -124,11 +124,16 @@ func (b *BpfRecorder) Run() error {
 		return errors.Wrap(err, "load in-cluster client")
 	}
 
-	listener, err := net.Listen("tcp", addr())
+	listener, err := net.Listen("unix", config.GRPCServerSocketBpfRecorder)
 	if err != nil {
 		return errors.Wrap(err, "create listener")
 	}
 
+	const rwa = os.FileMode(0o777)
```

Comment:
Does the file really need to be 777? (Why +x? Why +w others, can't we use a common groupid?)

Reply:
The problem is that we run the main container with (G)ID 65535 and the enricher/bpfrecorder as root. Means that the created socket file cannot be accessed by the rootless user. I think changing it to the rootless user may be better, I'll give this a try.